grakel/graph.py

```diff
@@ -1714,14 +1715,9 @@ def floyd_warshall(adjacency_matrix):
 
     dist = np.empty((n, n))
     # Initialization
-    for i in range(0, n):
-        for j in range(0, n):
-            if (i == j):
-                dist[i, j] = 0.
-            elif (adjacency_matrix[i, j] == 0):
-                dist[i, j] = float("Inf")
-            else:
-                dist[i, j] = adjacency_matrix[i, j]
+    dist = copy.deepcopy(adjacency_matrix)
```

Comment:
Hello there!
I've tried to run both old and new implementation.
The results are the same if the adjacency matric is a matrix with floats. But it breaks when the input are integers

I've tried both with 
x1 = np.random.randint(0, 10, (10, 10))
x2 = np.random.randint(0, 10, (10, 10)).astype(float)

x2 passes and returns the same results as the old. 
x1 raises overflow error

Reply:
Does np.inf solve this problem instead of float("Inf").
Can you please have a look?

Reply:
It is seems it is not easily solved.
https://stackoverflow.com/questions/56811682/how-can-i-convert-np-inf-to-an-integer-type

@Alessi0X can you please provide a workaround that supports integers in a new pull-request?